### PR TITLE
Add habits page + mobile first in styles.js

### PR DESCRIPTION
**Pull request by Sullynobugz**

### CODE REVIEW

File: components/AddHabits/AddHabits.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    if (inputValue.trim() === '') {
+      return;
+    }
+
+    setHabits([...habits, inputValue]);
+    setInputValue('');
+  };
+
+  const deleteHabit = (indexToDelete) => {
+    setHabits(habits.filter((_, index) => index !== indexToDelete));
```
Comment (Roland-Hufnagel) [inline components/AddHabits/AddHabits.js:19]: I recommend to save the habits as objects that have a unique 'id' and a 'title'. Then you can use the id for filtering.

File: components/AddHabits/AddHabits.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+          onChange={(e) => setInputValue(e.target.value)}
+          placeholder="Enter a new habit"
+        />
+        <button type="submit">Add Habit</button>
+      </form>
+      <ul>
+        {habits.map((habit, index) => (
+          <li key={index}>
+            {habit}
+            <button onClick={() => deleteHabit(index)}>Delete</button>
```
Comment (Roland-Hufnagel) [inline components/AddHabits/AddHabits.js:38]: please define the type: type="button"

File: styles.js [hunk 3]
```diff
@@ -10,5 +10,13 @@ export default createGlobalStyle`
...
     font-family: system-ui;
+    padding: 20px;
+  }
+
+  /* Mobile-first styles and media queries */
+  @media screen and (min-width: 768px) {
+    body {
+      /* Add styles for larger screens (e.g., tablets and desktops) */
+    }
   }
```
Comment (Roland-Hufnagel) [inline styles.js:21]: Do not write unused code.

**Review by Roland-Hufnagel — CHANGES_REQUESTED** [review]
Hi Bastian. This is a very good starting point. I left some comments. Please have a look at it.

**Review by NikRoe — APPROVED** [review]
LGTM 💪

**Pull request merged**